#for one-helm vehicles
#l:left wheel
#r:right wheel
#count clockwise,such as 1:front,2:rear or 1:front-left,2: front-right, 3:rear-right, 4:rear-left

float64 V
float64 Ang
